packages/liveblocks-react/src/index.ts

```diff
@@ -90,6 +90,11 @@ export {
   useUser,
 } from "./liveblocks";
 
-// Private APIs (for use in react-lexical only)
+// Private APIs
+
+// Used in @liveblocks/react-lexical
 export { getUmbrellaStoreForClient } from "./liveblocks";
 export { selectThreads } from "./umbrella-store";
+
+// Used in @liveblocks/react-ui and @liveblocks/react-lexical
+export { useMentionSuggestions } from "./use-mention-suggestions";
```

Comment:
Should we export this with an uglier name, to make it clear users should not rely on it? Just adding a `@private` field will still make it pop up in autocompletions I think.

Reply:
I was on the fence about doing too many changes in this area because of the TipTap work being done at the same time but there are already changes anyway, I'll go with the `@liveblocks/react/_private` convention 👍

Reply:
Done in https://github.com/liveblocks/liveblocks/pull/1992/commits/2ca30ffeda6232ee72079e957283db4b89bba940.